props/saber_fett263_buttons.h

```diff
@@ -5654,7 +5679,11 @@ SaberFett263Buttons() : PropBase() {}
   bool auto_lockup_on_ = false; // Battle Mode Lockup active
   bool auto_melt_on_ = false; // Battle Mode Melt/Drag active
   bool battle_mode_ = false; // Battle Mode active
+#ifndef FETT263_QUICK_SELECT_ON_BOOT  
```

Comment:
It's better to use #ifdef instead of #ifndef for these. (and swap the position of the two cases.)  Inverted logic takes a few extra moments to parse for anybody who reads the code.

Reply:
fixed